website/docs/guides/best-practices/how-we-build-our-metrics/semantic-layer-1-intro.md

```diff
@@ -13,10 +12,10 @@ Flying cars, hoverboards, and true self-service analytics: this is the future we
 
 ## Learning goals
 
-- ❓ Understand the **purpose and capabilities** of the **dbt Semantic Layer**.
-- 🧱 Familiarity with the core components of the dbt Semantic Layer — **semantic models and metrics** — and how they work together.
+- ❓ Understand the **purpose and capabilities** of the **dbt Semantic Layer**, particularly MetricFlow as the engine that powers it.
+- 🧱 Familiarity with the core components of MetricFlow — **semantic models and metrics** — and how they work together.
 - 🛠️ Hands-on **experience building** semantic models and metrics in dbt Cloud.
-- 🔁 Know how to **refactor** models into the Semantic Layer.
+- 🔁 Know how to **refactor** models for MetricFlow.
 - 🏅 Aware of new **best practices** to take maximum advantage of the Semantic Layer.
```

Comment:
this is prob a PMM question but I think the SL is referenced a few diff ways so do we want it to be dbt Semantic Layer or dbt Cloud Semantic Layer or Semantic Layer?